egs/multi_zh-hans/ASR/prepare.sh

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/env bash
+
+# fix segmentation fault reported in https://github.com/k2-fsa/icefall/issues/674
+export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python
+
+set -eou pipefail
+
+nj=16
```

Comment:
Looks like this argument is only used by `kespeech`. Shall we remove this argument or update other scripts to use this argument?